pkg/controller/nodeipam/ipam/cloud_cidr_allocator.go

```diff
@@ -159,17 +166,28 @@ func (ca *cloudCIDRAllocator) worker(stopChan <-chan struct{}) {
 func (ca *cloudCIDRAllocator) insertNodeToProcessing(nodeName string) bool {
 	ca.lock.Lock()
 	defer ca.lock.Unlock()
-	if ca.nodesInProcessing.Has(nodeName) {
+	if _, found := ca.nodesInProcessing[nodeName]; found {
 		return false
 	}
-	ca.nodesInProcessing.Insert(nodeName)
+	ca.nodesInProcessing[nodeName] = 0
+	return true
+}
+
+func (ca *cloudCIDRAllocator) retryNodeInProcessing(nodeName string) bool {
```

Comment:
rename `canRetry(nodeName string)`, this doesn't actually do the retry, it's a predicate